### DISCUSSION TITLE
global.sls and data.* gone missing??

### DISCUSSION THREAD

**Discussion by DrOnionMaster** [Security-Onion-Solutions/securityonion#12768] [General]
Hey everyone,

So here's my issue... when attempting to run soup to update from 2.4 to 2.5 I am running into the following error's:

    local:
        Data failed to compile:

    Pillar failed to render with the following messages:

    Specified SLS 'data.*' in environment 'base' is not available on the salt master

    Specified SLS 'global' in environment 'base' is not available on the salt master

I have checked for the global.sls file within /opt/so/saltstack/local/pillar/ and it does not exist.

[dronionmaster] > file *
backup:        directory
bpf:           directory
data:          directory
docker:        directory
elastalert:    directory
elasticfleet:  directory
elasticsearch: directory
firewall:      directory
global:        directory
healthcheck:   directory
idh:           directory
idstools:      directory
influxdb:      directory
kibana:        directory
kratos:        directory
logrotate:     directory
logstash:      directory
manager:       directory
minions:       directory
nginx:         directory
node_data:     directory
ntp:           directory
patch:         directory
pcap:          directory
redis:         directory
secrets.sls:   ASCII text
sensoroni:     directory
soc:           directory
soctopus:      directory
stig:          directory
strelka:       directory
suricata:      directory
telegraf:      directory
zeek:          directory
[dronionmaster] > file global.sls
global.sls: cannot open 'global.sls' (No such file or directory)

I'm not sure how I can get this file? or regenerate a new one? 

And I'm not sure what the 'data.*' file is?

**Comment by cm-ops** [comment]
Did you lose your connection during soup? What does this return - `sudo salt-call state.show_top`?

**Reply by DrOnionMaster** [reply]
I believe so, we ended up reverting to an old snapshot and are no longer seeing this error.

However the bigger issue is that we are getting hung during the update.

This is where we are getting hung:

`[INFO    ] Executing command '/usr/sbin/so-influxdb-manage' in directory '/root'`

I believe when I exited the update using ctrl-c last time it caused the error this post was originally about. Now that we are able to run `soup` again, we are still getting hung on this command. I don't want to exit and revert again, so I am looking for a solve for this new error.

**Reply by DrOnionMaster** [reply]
Moving discussion to #12783 as this is a new issue.
